controller/get_transaction.go

```diff
@@ -24,12 +25,12 @@ func GetTransactions(c *gin.Context) {
 
 	id := session.Get("id")
 
-	result := util.DB.Raw(`
-SELECT transactions.*, fromAccounts.social_security_number as from_social_security_number, fromAccounts.id, toAccounts.social_security_number as to_social_security_number, fromAccounts.phone as FromPhone,  toAccounts.phone as ToPhone 
+	result := util.DB.Raw(`SELECT transactions.*, transactions.transaction_datetime as TransactionDate, fromAccounts.social_security_number as from_social_security_number, fromAccounts.id, toAccounts.social_security_number as to_social_security_number, fromAccounts.phone as FromPhone,  toAccounts.phone as ToPhone 
 	FROM transactions
 	JOIN accounts AS fromAccounts ON fromAccounts.id = transactions.from_account_id
 	JOIN accounts AS toAccounts ON toAccounts.id = transactions.to_account_id
-	WHERE fromAccounts.id = ? OR toAccounts.id = ?`, id, id).Scan(&transactionResponses)
+	WHERE (fromAccounts.id = ? AND state = 'Successful') OR (toAccounts.id = ? AND state = 'Successful')
```

Comment:
![Screenshot 2023-10-09 at 23 22 10](https://github.com/Hugs-For-The-Bugs/bank-backend/assets/93491925/a469e0d4-e1cc-4edc-91af-7e4de2e45fd0)
Then  we will have  the failed for the sender I our return

Reply:
Does not need to change.

Reply:
Yeah, that's the idea...
I can see my failed transactions but not the failed ones from someone else...

Reply:
your query return with the records to the specific userID. And it can be included with failed transactions for that user. since you don't filter the records with the successful state.

Reply:
Exactly that's the goal 😀 
The user want's to see the failed transactions he did.
But not the transactions someone wanted to do for him and then they failed....

Reply:
Ok, I changed it.